doc: session special usage tip (#3304)
eggjs/egg: docs/source/en/core/cookie-and-session.md

};
```

> What you need to pay special attention to is that you need to avoid the following situations when setting session properties (which can cause field loss, See for details [koa-session source code](https://github.com/koajs/session/blob/master/lib/session.js#L37-L47)):

Reviewer: ```suggestion
Note:  Avoid the following situations when setting session properties (which can cause field loss, see more details at [koa-session source code](https://github.com/koajs/session/blob/master/lib/session.js#L37-L47)):
```
